code-scanning/soos-dast-scan.yml

```diff
@@ -24,18 +24,22 @@ jobs:
   soos:
     permissions:
       security-events: write # for uploading code scanning alert info
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status 
     name: SOOS DAST Scan
     runs-on: ubuntu-latest
     steps:
       - name: Run SOOS DAST Scan
-        uses: soos-io/soos-dast-github-action@5f8e2a1994d618e6ac9902e0f491fd1656b698e6
+        uses: soos-io/soos-dast-github-action@5f8c23ccf8366ea0a58deeb5c804e0524267df43
```

Comment:
v1.1.1 is `b524e2cfbc4f4a5733153a7e624f569913f6c6e9`. Can you check that sha again?

Reply:
yep, that's the right hash, thought I could put the next commit one's too since it had only a change on the readme.
I've just modified to the specific one where I've created the v.1.1.1

Reply:
Thanks. Putting release commit sha helps everyone keep track of the version.